### PR TITLE
Add issue templates

### LINKED ISSUE #6326: Improve issue templates
Numpy issue templates are just amazing. They have query parameters so you just fill the gaps

https://github.com/numpy/numpy/tree/main/.github/ISSUE_TEMPLATE

Let me know what you think of them

### PR DESCRIPTION
**Pull request by ferrine**
**What is this PR about?**
Closes #6326, #5119

## Docs / Maintenance
- Improves Issue Templates

### CODE REVIEW

**Comment by ferrine** [comment]
ready

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+- type: textarea
+  attributes:
+    label: "Describe the issue:"
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: "Reproduce the code example:"
```
Comment (michaelosthege) [inline .github/ISSUE_TEMPLATE/bug-report.yml]:
```suggestion
    label: "Reproduceable code example:"
```

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    description: >
+      A short code example that reproduces the problem/missing feature. It
+      should be self-contained, i.e., can be copy-pasted into the Python
+      interpreter or run as-is via `python myproblem.py`.
+    placeholder: |
+      import numpy as np
```
Comment (michaelosthege) [inline .github/ISSUE_TEMPLATE/bug-report.yml]:
```suggestion
      import pymc as pm
```

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    required: true
+
+- type: textarea
+  attributes:
+    label: "Error message:"
+    description: >
+      Please include full error message, if any.
+      If you are reporting a segfault please include a GDB traceback,
+      which you can generate by following
+      [these instructions](https://github.com/numpy/numpy/blob/main/doc/source/dev/development_environment.rst#debugging).
```
Comment (michaelosthege) [inline .github/ISSUE_TEMPLATE/bug-report.yml]:
```suggestion
      Please include full error message, if any.
```

File: .github/ISSUE_TEMPLATE/feature-request.yml [hunk 4]
```diff
@@ -0,0 +1,62 @@
+# This issue template was adapted from the NumPy project
+# under the BSD 3-Clause "New" or "Revised" License.
+# Copyright (c) 2005-2022, NumPy Developers.
+# All rights reserved.
+
+name: Feature request
+description: Check instructions for submitting your idea on the mailing list first.
```
Comment (michaelosthege) [inline .github/ISSUE_TEMPLATE/feature-request.yml]:
```suggestion
description: Make a specific, well-motivated proposal for a feature.
```

File: .github/ISSUE_TEMPLATE/feature-request.yml [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+  attributes:
+    value: >
+      If you're looking to request a new feature or change in functionality,
+      including adding or changing the meaning of arguments to an existing
+      function, please post your idea first on please create a discussion thread at
+      [Discussions](https://github.com/pymc-devs/pymc/discussions)
+      to validate it and bring attentino to it. After validation,
+      you can open this issue for a more technical developer discussion.
+      [Contributor Guide](https://github.com/pymc-devs/pymc/blob/main/CONTRIBUTING.md)
+      if you need more information.
```
Comment (michaelosthege) [inline .github/ISSUE_TEMPLATE/feature-request.yml]:
```suggestion
      function, please post your idea first as a [Discussion](https://github.com/pymc-devs/pymc/discussions)
      to validate it and bring attention to it. After validation,
      you can open this issue for a more technical developer discussion.
      Check the [Contributor Guide](https://github.com/pymc-devs/pymc/blob/main/CONTRIBUTING.md)
      if you need more information.
```

File: .github/ISSUE_TEMPLATE/feature-request.yml [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+    [Discussions](https://github.com/pymc-devs/pymc/discussions)
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: "Before"
+  descrition: >
+    Please fill the code snippet how did you workaround your problem or frequent use.
+    Leave empty if you see it impossible
```
Comment (michaelosthege) [inline .github/ISSUE_TEMPLATE/feature-request.yml]:
```suggestion
    Please fill the code snippet: How did you workaround your problem or frequent use?
    Leave empty if you found no workaround.
```

**Comment by ferrine** [comment]
Fixed issues raised by @michaelosthege

**Review by michaelosthege — APPROVED** [review]

**Pull request merged**